src/main/java/com/spotinst/sdkjava/example/ElastigroupUsageExampleAzure.java

```diff
@@ -1,8 +1,7 @@
 package com.spotinst.sdkjava.example;
```

Comment:
Create separate folder for each example class.

Reply:
created for aws, azure, gcp